packages/cli/fixtures/test.sh

```diff
@@ -1,134 +0,0 @@
-#!/bin/bash
```

Comment:
Sure this file should go away? Did these tests move somewhere else?

Reply:
They've been ported to `src/__tests__/commands/Generate.test.ts`